run_unit_tests.py

```diff
@@ -0,0 +1,78 @@
+'''
+This helper script for Seattle testbed, https://seattle.poly.edu/,
+ * downloads Seattle testbed's sources from GitHub,
+ * builds them using Seattle's build scripts, and finally
+ * runs Seattle's unit tests.
+
+ Find documentation for Seattle's build scripts and unit test framework at 
+   https://seattle.poly.edu/wiki/BuildInstructions
+   https://seattle.poly.edu/wiki/UnitTestFramework
+
+ The list of components (i.e. source repositories) for which the 
+ tests are run is configurable via the repolist variable, see below.
+
+ Since other testbeds use the Seattle codebase, we make the testbed 
+ name (and thus repo address) configurable too:
```

Comment:
Documentation says that `repolist` and `testbed` are configurable.  It appears that they must modified directly in the code.  Does it make sense to make them configurable from the command-line?